engine/table/src/main/java/io/deephaven/engine/table/impl/perf/UpdatePerformanceTracker.java

```diff
@@ -251,6 +314,26 @@ public long getIntervalDurationNanos() {
 
     @NotNull
     public QueryTable getQueryTable() {
-        return MemoryTableLogger.maybeGetQueryTable(tableLogger);
+        // todo: should this interface still exist? Why QueryTable? If so, should we return blink, derived, or
```

Comment:
Maybe we `@Deprecate` it and return the derived table for short-term backwards compatibility?